src/topology/algebra/ordered.lean

```diff
@@ -1866,6 +1906,22 @@ lemma intermediate_value_Icc' {a b : α} (hab : a ≤ b) {f : α → β} (hf : c
   Icc (f b) (f a) ⊆ f '' (Icc a b) :=
 is_preconnected_Icc.intermediate_value (right_mem_Icc.2 hab) (left_mem_Icc.2 hab) hf
 
+/-- A continuous function which tendsto `at_top` `at_top` and to `at_bot` `at_bot` is surjective. -/
+lemma surjective_of_continuous {f : α → β} (hf : continuous f) (h_top : tendsto f at_top at_top)
```

Comment:
Please add a version with `tendsto f at_top at_bot` and `tendsto f at_bot at_top`. I think that `:= @surjective_of_continuous (order_dual α) β ...` should work as a proof.